<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Select</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Really_Crappy_about_Scaled Down.png</normaloff>Really_Crappy_about_Scaled Down.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="FilePath">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>The Path to the Selected File</string>
   </property>
   <property name="text">
    <string>C:\</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FileSelector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>Open a File Selection box</string>
   </property>
   <property name="text">
    <string>Select File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
